test/Graphics/Drawable.test.cpp

```diff
@@ -1,9 +1,49 @@
 #include <SFML/Graphics/Drawable.hpp>
+#include <SFML/Graphics/RenderTexture.hpp>
 
+#include <doctest/doctest.h>
+
+#include <GraphicsUtil.hpp>
 #include <type_traits>
 
 static_assert(!std::is_constructible_v<sf::Drawable>);
 static_assert(!std::is_copy_constructible_v<sf::Drawable>);
 static_assert(std::is_copy_assignable_v<sf::Drawable>);
 static_assert(!std::is_nothrow_move_constructible_v<sf::Drawable>);
 static_assert(std::is_nothrow_move_assignable_v<sf::Drawable>);
+
+class DrawableTest : public sf::Drawable
+{
+public:
+    void draw(sf::RenderTarget&, const sf::RenderStates&) const override
+    {
+        m_called += 1;
+    }
+
+    int callCount() const
+    {
+        return m_called;
+    }
+
+private:
+    mutable int m_called{};
+};
+
+TEST_CASE("[Graphics] sf::Drawable"  * doctest::skip(skipDisplayTests))
+{
+    SUBCASE("Construction")
+    {
+        const DrawableTest drawableTest;
+        CHECK(drawableTest.callCount() == 0);
+    }
+
+    SUBCASE("draw()")
+    {
+        const DrawableTest drawableTest;
+        sf::RenderTexture  renderTexture;
+        CHECK(renderTexture.create({32, 32}));
```

Comment:
If this `.create` call fails, are the following tests still valid or do they become UB?

Reply:
Oh, you're right.

Reply:
It should be well defined because we aren't actually manipulating anything.

We might not even need to call create? Nothings being drawn, and nothing is being changed. It just needed a sf::RenderTarget that exists in memory (to be passed by reference).

https://github.com/SFML/SFML/blob/ca5ca650040580a15890136b953d82d51c4077a0/include/SFML/Graphics/RenderTarget.hpp#L249

https://github.com/SFML/SFML/blob/ca5ca650040580a15890136b953d82d51c4077a0/src/SFML/Graphics/RenderTarget.cpp#L247